geometry_msgs/Point point
float64 radius
float64 lenght
